clients/cloud/python/README.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+
+Produce messages to and consume messages from Confluent Cloud using Confluent Python Client for Apache Kafka
+
+
+# Prerequisites
+
+* Install Confluent's Python Client for Apache Kafka: https://github.com/confluentinc/confluent-kafka-python
```

Comment:
Does this work for any version of the client? If not, we need to specify a version (old versions of librdkafka had different configurations, IIRC).

Reply:
@edenhill 

1. Can you please advise if there are version caveats/constraints that we need to call out?  
2. If there are caveats, in general it may make sense to capture those NOT in this README but rather in the README in https://github.com/confluentinc/confluent-kafka-python , which should be the source of truth